extensions/wikia/Search/classes/Services/SeriesEntitySearchService.php

```diff
@@ -38,18 +38,28 @@ protected function prepareQuery( string $query ) {
 			$select->createFilterQuery( 'type' )->setQuery( '+(article_type_s:' . static::SERIES_TYPE . ')' );
 		}
 
-		$dismax->setQueryFields( implode( ' ', [
-			'title_em^8',
-			'titleStrict',
-			$this->withLang( 'title', $slang ),
-			$this->withLang( 'redirect_titles_mv', $slang ),
-		] ) );
-		$dismax->setPhraseFields( implode( ' ', [
-			'title_em^8',
-			'titleStrict^8',
-			$this->withLang( 'title', $slang ) . '^2',
-			$this->withLang( 'redirect_titles_mv', $slang ) . '^2',
-		] ) );
+		$dismax->setQueryFields(
+			implode(
+				' ',
+				[
+					'title_em^8',
+					'titleStrict',
+					$this->withLang( 'title', $slang ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)